src/Collection/Sanitize.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Lean\Endpoints\Collection;
+
+/**
+ * Class that provides some helper functions to Sanitize elements.
+ *
+ * @package Lean\Endpoints
+ */
+class Sanitize {
+
+	/**
+	 * Function that iterates over an array to Sanitize the value of the array
+	 * in secure text values.
+	 *
+	 * @param array $data The data to be Sanitized.
+	 * @return array The cleaned data.
+	 */
+	public static function text_array( array $data = [] ) {
+		$map = function( $item ) {
+			return sanitize_text_field( $item );
+		};
+		return array_map( $map, $data );
```

Comment:
Thanks for the tip, and technically `sanitize_text_field` is a named clousre isn`t?

> On Mar 31, 2016, at 5:07 AM, Caridad Zapatero Contreras <notifications@github.com> wrote:
> 
> In src/Collection/Sanitize.php <https://github.com/moxie-leean/wp-endpoints-collection/pull/2#discussion_r58036339>:
> 
> > + * @package Lean\Endpoints
> > + */
> > +class Sanitize {
> > +
> > +	/**
> > +	 * Function that iterates over an array to Sanitize the value of the array
> > +	 * in secure text values.
> > +	 *
> > +	 * @param array $data The data to be Sanitized.
> > +	 * @return array The cleaned data.
> > +	 */
> > +	public static function text_array( array $data = [] ) {
> > +		$map = function( $item ) {
> > +			return sanitize_text_field( $item );
> > +		};
> > +		return array_map( $map, $data );
> Do we need a closure here? Doesnt this work?
> return array_map( 'sanitize_text_field', $data );
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-endpoints-collection/pull/2/files/86356df199c5bac0bd0cfbda93ff685a610a4205#r58036339>